pylint_odoo/checkers/no_modules.py

```diff
@@ -213,8 +221,25 @@ class NoModuleChecker(BaseChecker):
         }),
     )
 
+    def visit_binop(self, node):
```

Comment:
Why use a `visit_*` method without a `@utils.check_messages(` decorator.
Should we change the logic for this method instead of a `visit_callfunc`?